src/test/resources/com/puppycrawl/tools/checkstyle/grammars/javadoc/javadocTags/InputSpaceBeforeDescriptionInBlockJavadocTags.javadoc

```diff
@@ -38,16 +38,4 @@ include
 
 * @customTag
      
-     DESCRIPTION
-
-* @exception
-      Exception     Exception-description
-      spanning multiple lines
-
-* @param
-      ParamName
-      Param-description
-
-* @throws
-     Exception     Exception-description
-         spanning multiple lines
```

Comment:
These input cases should be added in #4980 .

Reply:
This is because:

$ cat test.javadoc
```
* @exception
      Exception     Exception-description
      spanning multiple lines
```

java -jar master-8.2-all.jar -j test.javadoc
```
JAVADOC -> JAVADOC [0:0]
|--LEADING_ASTERISK -> * [0:0]
|--WS ->   [0:1]
|--JAVADOC_TAG -> JAVADOC_TAG [0:2]
|   |--EXCEPTION_LITERAL -> @exception [0:2]
|   |--NEWLINE -> \r\n [0:13]
|   |--WS ->        [1:0]
|   `--DESCRIPTION -> DESCRIPTION [1:6]
|       |--TEXT -> Exception     Exception-description [1:6]
|       |--NEWLINE -> \r\n [1:41]
|       `--TEXT ->       spanning multiple lines [2:0]
`--EOF -> <EOF> [2:29]
```

As can be seen `CLASS_NAME` which has been made mandatory for the corresponding javadoc tag is not being recognized in the above example and hence results in parse errors. This should be resolvable by adding `NEWLINE` to corresponding lexer modes but of course one can't be sure of it until a solution is attempted.